src/TwigRendererFactory.php

```diff
@@ -72,6 +73,14 @@ public function __invoke(ContainerInterface $container)
             $environment->addExtension(new TwigExtensionDebug());
         }
 
+        // Add user defined extensions
+        $extensions = isset($config['helpers']['twig']) && is_array($config['helpers']['twig'])
```

Comment:
This is the only part of the patch I don't like. Currently, the various items under the `templates` configuration key are generic enough that they can work with any of the renderers, making switching relatively easy. The two that offer more configuration are Twig and zend-view:

- Twig adds `cache_dir`, `assets_url`, and `assets_version`
- zend-view adds `layout` and `map`.

zend-view helpers are typically managed by the zend-view HelperPluginManager, and, as such, have their own top-level key as a sibling to `templates`. My thought at this time is to do similarly here:

```php
return [
    'templates' => [
        /* ... */
    ],
    'twig' => [
        'helpers' => [ /* ... */ ],
    ],
];
```

I'd also argue that the `cache_dir` and `assets_*` keys should move there.

I'm trying to get a 0.3.0 release of this out today; I hope you don't mind if I make those changes?